code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -351,6 +359,10 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	smoketime = 7200
 	chem_volume = 30
 
+/obj/item/clothing/mask/smokable/cigarette/cigar/havana/New()
+	..()
+	reagents.add_reagent(/datum/reagent/nicotine, 10))

```

Comment:
With the super call [..()], this will have 17 units of nicotine in it.

Reply:
Ah. So, would I just want to remove the ..() and it'll be 10 units, or would I want to just adjust the amount in the add_reagent?

Reply:
You want to adjust the amount you're adding in that add_reagent call. Removing ..() tends to have unforeseen consequences.